metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -243,6 +235,9 @@ def _build_metrics_output_node(
                         aggregated_measures_node=aggregated_measures_node,
                     )
                 )
+
+        assert len(compute_metrics_nodes) > 0, "ComputeMetricsNode was not properly constructed"
```

Comment:
Should this assertion have existed before or is this now needed due to how the constraints are being bundled?

Reply:
just a sanity check that shouldve existed before, something wouldve went wrong if `len == 0`.